Client/Ecosia/Extensions/HomepageViewController+Ecosia.swift

```diff
@@ -82,26 +82,6 @@ extension HomepageViewController: NTPLibraryDelegate {
     }
 }
 
-extension HomepageViewController: NTPConfigurableNudgeCardCellDelegate {
-    func nudgeCardRequestToPerformAction(for cardType: HomepageSectionType) {
-        switch cardType {
-        case .newsletterCard:
-            BrazeService.shared.logCustomEvent(.newsletterCardClick)
```

Comment:
I wonder whether Braze's custom events, namely `newsletterCardClick` should be removed or kept as an example since it's the only one 🤔 Can we actually have empty enums? If so that might be cleaner.

Reply:
This one I left it there on purpose as I liked the idea of having the custom event.
I added an `empty` value to overcome this but keep the enum. Maybe it's a good compromise